### DISCUSSION TITLE
List with equal width columns

### DISCUSSION THREAD

**Discussion by smartm0use** [fomantic/Fomantic-UI#1732] [Elements (button, image, label, loader, ...)] [answered]
Hello, I'm using [horizontal list](https://fomantic-ui.com/elements/list.html#horizontal) like this:

![image](https://user-images.githubusercontent.com/5399265/97695141-dde9a380-1aa3-11eb-8149-b782aeb38c72.png)

but I would like to have equal width for every item element, like this:

![image](https://user-images.githubusercontent.com/5399265/97695236-07a2ca80-1aa4-11eb-8a87-d4a70e018941.png)

How can I do this without using `table`?

Here is my playground: https://jsfiddle.net/q6ok4j5e/

**Comment by smartm0use** [comment, marked as answer]
Any idea?

**Reply by ColinFrick** [reply]
Instead of a horizontal list you could use a grid https://fomantic-ui.com/collections/grid.html

**Comment by lubber-de** [comment]
Or use the same approach as for form fields (where a `equal width` exists (but only for one row)
https://jsfiddle.net/lubber/ts7q6uda/3/
```css
.ui.horizontal.list{
 display: flex;
}

.ui.horizontal.list .item {
  width: 100%;
  flex: 1 1 auto;
}
```

**Comment by smartm0use** [comment]
Thank you for your replies. I think using `grid` is the right solution in my case.
